apps/site/src/components/Sticker/Stickers/HeartSticker/HeartSticker.tsx

```diff
@@ -1,21 +1,26 @@
 import HeartEmoji from "@/assets/images/heart_emoji.png";
 import BaseSticker from "../../BaseSticker";
 import { fastShake } from "@/components/animation";
+import React from "react";
 
-export default function HeartSticker({
-	style,
-}: {
-	style?: object | undefined;
-}) {
-	return (
-		<div style={{ ...style }}>
-			<BaseSticker
-				imageSrc={HeartEmoji.src}
-				alt="heart emoji sticker"
-				height={150}
-				width={150}
-				{...fastShake}
-			/>
-		</div>
-	);
+interface HeartStickerProps {
+	children: React.ReactNode;
+	position?: "top-left" | "top-right" | "bottom-left" | "bottom-right";
 }
+
+const HeartSticker = React.forwardRef<
```

Comment:
I believe the ref from `forwardRef` is unused. Is there another reason it is used?

Reply:
I forgot to pass the ref, but I don't actually think it's needed here. I'll remove it.